crates/supervisor/core/src/syncnode/node.rs

```diff
@@ -244,45 +248,75 @@ impl ManagedNode {
     ///
     /// Analyzes the event content and takes appropriate actions based on the
     /// event fields.
-    /// TODO: Call relevant DB functions to update the state
-    fn handle_managed_event(event_result: Option<ManagedEvent>) {
+    async fn handle_managed_event(
+        event_tx: &mpsc::Sender<NodeEvent>,
+        event_result: Option<ManagedEvent>,
+    ) {
         match event_result {
             Some(event) => {
-                debug!(target: "managed_node", ?event, "Handling ManagedEvent");
+                debug!(target: "managed_node", %event, "Handling ManagedEvent");
```

Comment:
```suggestion
                debug!(target: "managed_node", ?event, "Handling ManagedEvent");
```
here is a place where it makes most sense to just use the debug format, since these are debug logs, and the implementation that was done for display, Is basically the debug format, so we can just use debug format here and remove the display impl